src/DynamicData/List/ChangeSet.cs

```diff
@@ -55,6 +57,8 @@ public ChangeSet([NotNull] IEnumerable<Change<T>> items)
             {
                 Add(change, true);
             }
+
+            _equalityComparer = equalityComparer;
```

Comment:
I would add a null colesing operator and default to the default equality operator for the null case

Reply:
I see! And I should probably do the same in the other constructor, just above. That would also mean it's never null and I could simplify the rest of the code.

Reply:
Done where it is now relevant :)